test/controllers/teams_users_controller_test.rb

```diff
@@ -0,0 +1,71 @@
+require 'test_helper'
```

Comment:
Please include the test_helper.rb in test directory.

Create the test DB using the below command:
rake db:create RAILS_ENV=test

Run the tests using the below command:
rake test test/controllers/teams_users_controller_test.rb